File: concordion/src/test/java/base/DriverFixture.java
package base;

import org.junit.After;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class DriverFixture {
    
    private WebDriver driver;

	protected WebDriver getDriver() {
        driver = new FirefoxDriver();
        return driver;
    }
    
    @After
    public void close() {
    	driver.close();
    }
}

File: concordion.net/src/concordion.spec/Google/DriverFixture.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;

namespace Concordion.Spec.Google
{
    public class DriverFixture : IDisposable
    {
        protected readonly IWebDriver WebDriver = new FirefoxDriver();

        public void Dispose()
        {
            // TODO: unsure why this isn't getting called - tb, feb 2012
            WebDriver.Quit();
        }
    }
}